docs/guides/nextjs.mdx

```diff
@@ -44,6 +44,10 @@ const postData = atom((get) => {
 })
 ```
 
+## Provider
+
+Because of the nature of SSR, your app can have more instances existing in JS memory in the same time. You need to wrap your app inside a `Provider` (view more details [in the Core section](../api/core.mdx)) so that each instance has its own state and will not interfere with previous values from a common Provider (provider-less mode).
```

Comment:
```suggestion
Because of the nature of SSR, your app can have more instances existing in JS memory in the same time. You need to wrap your app inside a `Provider` (view more details [in the Core section](../api/core.mdx)) so that each instance has its own state and will not interfere with previous values from a default store (provider-less mode).
```

to make a little more precise. Though, we don't often use the term "store" in docs yet.